### PR TITLE
Refs #7745: Deploy client cert bundle specifically for use by the Capsul...

### PR DESCRIPTION
**Pull request by ehelms**
...e.

Note this is a bundle since that is required by the reverse proxy
feature being added to the Capsule.

### CODE REVIEW

File: manifests/capsule.pp [hunk 1]
```diff
@@ -10,31 +10,67 @@
 # $certs_tar::               path to tar file with certs to generate. REQUIRED
 #
```
Comment (dustints) [inline manifests/capsule.pp]: comment for `$generate` argument needed

File: manifests/capsule.pp [hunk 2]
```diff
@@ -10,31 +10,67 @@
...
+  }
 
-  # until we support again pushing the cert rpms to the Katello,
-  # make sure the certs_tar path is present
-  validate_present($certs_tar)
-  validate_present($capsule_fqdn)
+  if $generate {
+    # until we support again pushing the cert rpms to the Katello,
+    # make sure the certs_tar path is present
+    validate_present($certs_tar)
```
Comment (dustints) [inline manifests/capsule.pp]: when installing katello bin/katello-installer exec in katello installer, validation fails.

**Comment by ehelms** [comment]
@dustints @stbenjam updated - this is still required for the reverse proxy (which I will be updating shortly) capsule PR but this can in theory be a stand alone change.

**Comment by stbenjam** [comment]
Tested & works, ACK from me!

**Pull request merged**